### ISSUE TITLE
[Docs]: Include note in Docker docs about enabling remote management

### ISSUE THREAD

**Issue by joostdecock** [smallstep/docs#160]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to document this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Affected area/feature

When running in a Docker environment, you can't do all that much without enabling remote management.

From what I read on Discord, this seems to be a common issue for people starting out with the CA on Docker, so I suggest to add a note in the [Docker info](https://hub.docker.com/r/smallstep/step-ca).

Something like:

> To enable remote management, include `"enableAdmin": true` in the authority object of `ca.json`.

Hat tip to @hslatman for explaining this to me.

**Comment by joostdecock** [comment]
For those who face the same problem, you can patch your config with this one-liner:

```
echo "$( jq '.authority.enableAdmin = true' ca.json)" > ca.json
```

**Comment by joostdecock** [comment]
It might also be good to include a note in the Docker docs that `step` is the default super admin user.
I think especially because the default provisioner is named `admin`, I somehow assumed that would be the username to use (it's a very typical admin username after all). 

I was wrong, and yes, it's documented at https://smallstep.com/docs/step-ca/provisioners#remote-provisioner-management but it would still save idiots like me a lot of searching :)

**Comment by hslatman** [comment]
Adding this PR here for reference: https://github.com/smallstep/certificates/pull/1075. This makes it a bit easier to initialize a CA with Remote Management enabled. 

A quick follow-up on that one: https://github.com/smallstep/certificates/pull/1140. This will print the `step` super admin subject and the (default) admin provisioner when initializing the CA.

**Comment by tashian** [comment]
Thanks for your work on this one @hslatman.

I've also opened smallstep/certificates#1223 so that the remote management password isn't the same as the CA private key password by default, and to print out the provisioner password when initializing the CA in Docker.

I have a draft of an updated Docker Hub README that reflects all of these changes. In the `README`, remote management is now enabled by default.  Updating the Docker Hub README is blocked on a release of `step-ca`, because we need your updated entrypoint script with the init vars in the Docker image.

**Comment by tashian** [comment]
Closing this, it's addressed in 0.23.1. Thank you @joostdecock for raising this!
